tests/test_user.py

```diff
@@ -96,7 +121,19 @@ def set_anonymous_comment(request):
     )
 
 
-@pytest.mark.usefixtures('set_user_client', 'set_user_client2', 'set_board', 'set_articles')
+@pytest.fixture(scope='class')
+def set_realname_comment(request):
+    """set_realname_articles 먼저 적용"""
```

Comment:
1. `s` 오타 있습니다.
2. line 99처럼 `set_user_with_kaist_info`도 명시하면 좋겠습니다.
```suggestion
    """set_realname_article, set_user_with_kaist_info 먼저 적용"""
```